docs/configuration/settings.rst

```diff
@@ -0,0 +1,586 @@
+
+.. code-block:: ini
+
+    # cliquet.backoff = 10
+    cliquet.retry_after_seconds = 30
+
+Similarly, the end of service date can be specified by using these settings.
+
++---------------------------------------+--------------------------------------------------------------------------+
+| Setting name                          | What does it do?                                                         |
++=======================================+==========================================================================+
+| cliquet.eos ``None``                  | The End of Service Deprecation date. If the date specified is in the     |
+|                                       | future, an alert will be sent to clients. If it’s in the past, the       |
+|                                       | service will be declared as decomissionned. If set to `None`, no End of  |
+|                                       | Service information will be sent to the client.                          |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.eos_message ``None``          | The End of Service message. If set to `None`, no End of Service message  |
+|                                       | will be sent to the clients.                                             |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.eos_url ``None``              | The End of Service information URL.                                      |
++---------------------------------------+--------------------------------------------------------------------------+
+
+.. code-block:: ini
+
+    cliquet.eos = 2015-01-22
+    cliquet.eos_message = "Client is too old"
+    cliquet.eos_url = http://website/info-shutdown.html
+
+
+Enabling or disabling endpoints
+===============================
+
+It is possible to deactivate specific resources operations, directly in the
+settings.
+
+To do so, a setting key must be defined for the disabled resources endpoints::
+
+    'cliquet.{endpoint_type}_{resource_name}_{method}_enabled'
+
+Where:
+- **endpoint_type** is either collection or record;
+- **resource_name** is the name of the resource (by default, *Cliquet* uses
+  the name of the class);
+- **method** is the http method (in lower case): For instance ``put``.
+
+For instance, to disable the PUT on records for the *Mushrooms* resource, the
+following setting should be declared in the ``.ini`` file:
+
+.. code-block:: ini
+
+    # Disable article collection DELETE endpoint
+    cliquet.collection_article_delete_enabled = false
+
+    # Disable mushroom record PATCH endpoint
+    cliquet.record_mushroom_patch_enabled = false
+
+Activating the flush endpoint
+=============================
+
+When using Kinto in development mode, it might be helpful to have a way to
```

Comment:
Consider:
> The Flush endpoint is used to flush (completely remove) all data from the database backend. While this can be useful during development, it's too dangerous to leave on by default, and must therefore be enabled explicitly.